rna/README.md

```diff
@@ -72,4 +72,22 @@ The `samples` table would then contain, for each sample:
 - [other samtools output files]
 - featureCounts_files (Array of STRINGS: paths to featureCounts output files)
 
-All of your workflow outputs will be stored in your Terra workspace bucket and the `sample` entity table will contain the paths to the outputs.
\ No newline at end of file
+All of your workflow outputs will be stored in your Terra workspace bucket and the `sample` entity table will contain the paths to the outputs.
```

Comment:
I think this paragraph belongs before the step-by-step instructions:

Thus if you set each workflow to run from the sample entity and set the outputs to update the sample entity, you can run a progression of workflows. All of your workflow outputs will be stored in your Terra workspace bucket and the `sample` entity table will contain the paths to the outputs.